source/user/howto/add-to-DEEP-marketplace.rst

```diff
@@ -0,0 +1,62 @@
+Add you model/service to the DEEP marketplace
+=============================================
+
+This document describes how to add your trained service or model to the DEEP marketplace.
+
+Creating the Github repositories
+--------------------------------
+
+You model must have a repository to host the code and a repository to host the Dockerfiles. Both these repositories can he hosted under your personal Github account. Naming conventions are that the Docker repo name is the same as the code repo name with the prefix ``DEEP-OC-``.
+
+A typical example of this can be:
+
+* `deephdc/image-classification-tf <https://github.com/deephdc/image-classification-tf>`_ - The Github repo hosting the code of an image classfication model.
+* `deephdc/DEEP-OC-image-classification-tf <https://github.com/deephdc/DEEP-OC-image-classification-tf>`_ - The Github repo hosting the Dockerfiles of the image classification model.
+
+In case you are only developing a service based on an already existing model (like for example developing an animal classifier based on the image-classification-tf module) you only need to create the Docker repo.
+
+The code repo
+^^^^^^^^^^^^^
+
+This is the repo containing the code of your model. If you are adding a service (ie. a trained model) the weights of the trained model must be stored in your personal storage services (e.g Google Drive, Dropbox, etc). A few MUSTs your code has to comply with in order to ensure compatibility and ease of use:
```

Comment:
Maybe we should state something different here:

"If you are adding a service (ie. a trained model) the weights of the trained model must be stored in a location accessible over a network connection, so that your container can download it upon creation. A few MUSTs your code has to comply with in order to ensure compatibility and ease of use:

Reply:
Done